packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -555,7 +570,8 @@ export const announceDefaults = ({
     return r.path.startsWith(`/${defaultExtensionId}`) && r.meta?.entryPoint === true
   })
   if (!route) {
-    route = store.getters.getNavItemsByExtension(defaultExtensionId)[0]?.route
+    route = getExtensionNavItems({ extensionRegistry, appId: defaultExtensionId })[0]
+      ?.route as RouteRecordNormalized
```

Comment:
Why do we need the cast here? Can't we fix the type?

Reply:
I knew you were going to remark this 😂 I guess so, but then we need to change it in the appInfo type and in all apps as well.

Reply:
not sure, but I figure there might be a better place for this cast at least

Reply:
ok, the place is good - but I'm happier with widening the type of the route var instead of casting the `.route` prop to something it isn't